### DISCUSSION TITLE
Cloudflare Installation method

### DISCUSSION THREAD

**Discussion by KennethEgholm** [teslamate-org/teslamate#3124] [Ideas]
I am struggling a little with getting my TeslaMate system exposed through CloudFlare (tunnels) ..
It would be an awesome if somebody knows how to set this up could make a guide - and maybe even have it shown as an installation option.

Thanks in advance, Kenneth Egholm

**Comment by xiaoyaoz1** [comment]
> I am struggling a little with getting my TeslaMate system exposed through CloudFlare (tunnels) .. It would be an awesome if somebody knows how to set this up could make a guide - and maybe even have it shown as an installation option.
> 
> Thanks in advance, Kenneth Egholm

「映射的时候使用127.0.0.1:4000或者3000端口到域名应该就可以了，可以使用免费的域名pp.ua或者刷的免费二级域名，我看你应该是在国外也可以注册一个甲骨文免费的服务器，铺设到公网，这样就省去cloudflare的隧道了，直接设置ip访问即可，域名也可以用dynv6注册一个简单的，v6.army尾缀很方便记忆，希望能帮到你，我的英文不好，你可以试着使用chatgpt翻译一下」
- - - - - - - - - - - - - - -


> 我有点挣扎，通过CloudFlare（隧道）暴露我的TeslaMate系统......如果有人知道如何设置这个，可以做一个指南，甚至可能将其显示为安装选项，那就太棒了。
> 
> 提前感谢，Kenneth Egholm

映射的时候使用127.0.0.1:4000或者3000端口到域名应该就可以了，可以使用免费的域名pp.ua或者刷的免费二级域名，我看你应该是在国外也可以注册一个甲骨文免费的服务器，铺设到公网，这样就省去cloudflare的隧道了，直接设置ip访问即可，域名也可以用dynv6注册一个简单的，v6.army尾缀很方便记忆，希望能帮到你，我的英文不好，你可以试着使用chatgpt翻译一下
